.travis.yml

```diff
@@ -2,9 +2,27 @@ language: python
 
 python: 2.7
 
-install: "pip install flake8"
+addons:
+  - firefox: "42.0"
+
+before_script:
+- "export DISPLAY=:99.0"
+- "sh -e /etc/init.d/xvfb start"
+- sleep 3 # give xvfb some time to start
+
+install:
+  - sudo apt-get update
+  - sudo apt-get install firefox
+  - pip install flake8
+  - pip install cryptography
```

Comment:
We shouldn't need to install cryptography separately if we're doing an install rather than a develop.